src/layouts/SiteDashboard/SiteDashboard.tsx

```diff
@@ -193,9 +200,67 @@ export const SiteDashboard = (): JSX.Element => {
           {/* Right column */}
           <Box w="40%">
             <VStack spacing="1.25rem">
+              {/* Site launch display card */}
+              {isUserUsingSiteLaunchFeature(siteName) &&
+                siteLaunchStatus !== "LAUNCHED" && (
+                  <Box w="100%">
+                    <DisplayCard variant="content">
+                      <Skeleton isLoaded={!isSiteLaunchLoading}>
+                        <DisplayCardHeader
+                          button={
+                            <Link
+                              variant="link"
+                              textStyle="subhead-1"
+                              color="text.title.brand"
+                              marginRight="0.75rem"
+                              href="./siteLaunchPad"
+                            >
+                              {siteLaunchStatus === "NOT_LAUNCHED" && (
+                                <Text>Go to Launchpad</Text>
+                              )}
+                              {siteLaunchStatus ===
+                                "CHECKLIST_TASKS_PENDING" && (
+                                <Text>Visit launchpad</Text>
+                              )}
+                              {siteLaunchStatus === "LAUNCHING" && (
+                                <Text>Check Status</Text>
+                              )}
```

Comment:
not an issue on my end, will let nat decide if this should be fixed but the text looks abit cramped